crosschain-resolver/README.md

```diff
@@ -58,28 +58,28 @@ Storage verificaton can only verify the data of l2. When the function result nee
     }
 ```
 
+## Deploying (Sepolia)
 
+Before deploying l1 contracts, deploy l2 contracts on https://github.com/ensdomains/ens-contracts
```

Comment:
Can you be more specific?

Reply:
What needs to be more specific? The exact command to deploy is specified at line 66-68